JournalMonitor/JournalMonitor.cs

```diff
@@ -331,6 +331,21 @@ public static List<Event> ParseJournalEntry(string line)
                                     events.Add(new ExplorationPromotionEvent(timestamp, rating) { raw = line });
                                     handled = true;
                                 }
+                                else if (data.ContainsKey("Federation"))
+                                {
+                                    Superpower superpower = Superpower.FromName("Federation");
+                                    data.TryGetValue("Federation", out val);
+                                    FederationRating rating = FederationRating.FromRank((int)(long)val);
```

Comment:
Why the double cast here and below?

Reply:
mirroring other code snippets from JournalMonitor.cs. I'm guessing that jgm wasn't sure whether he'd be pulling a number or a string?